libs/SalesforceSDKCore/SalesforceSDKCore/Classes/OAuth/SFOAuthOrgAuthConfiguration.m

```diff
@@ -54,6 +55,15 @@ - (BOOL)useNativeBrowserForAuth {
     return [self.authConfigDict[kAuthConfigMobileSDKKey][kAuthConfigUseNativeBrowserKey] boolValue];
 }
 
+- (BOOL)shareBrowserSession {
+    if ([self.authConfigDict[kAuthConfigMobileSDKKey] objectForKey:kAuthConfigShareBrowserSession] != nil) {
+        return [self.authConfigDict[kAuthConfigMobileSDKKey][kAuthConfigShareBrowserSession] boolValue];
+    } else {
+        // return false which means prompt=login being appended to login url
+        return false;
```

Comment:
why's the fallback different than the internal fork?

Reply:
The fork had a different fallback for iOS and Android.
I wanted the fallback to be prompt=login.
Also I checked with Maggie and she said "it doesn't matter today as servers are all upgraded to 242+, and server side will return false by default".

Reply:
👍 thanks!